src/gpu_sysman.c

```diff
@@ -1259,20 +1265,26 @@ static bool gpu_mems_bw(gpu_device_t *gpu) {
     gpu->membw_count = mem_count;
   }
 
+  /* names based on network metrics:
+   * https://opentelemetry.io/docs/specs/semconv/system/hardware-metrics/#hwnetwork---network-adapter-metrics
+   */
   metric_family_t fam_ratio = {
       .help = "Average memory bandwidth usage ratio (0-1) over query interval",
-      .name = METRIC_PREFIX "memory_bw_ratio",
+      .name = METRIC_PREFIX "memory.bandwidth.utilization",
       .type = METRIC_TYPE_GAUGE,
+      .unit = "1",
   };
   metric_family_t fam_rate = {
       .help = "Memory bandwidth usage rate (in bytes per second)",
-      .name = METRIC_PREFIX "memory_bw_bytes_per_second",
+      .name = METRIC_PREFIX "memory.io.rate",
       .type = METRIC_TYPE_GAUGE,
+      .unit = "By/s",
   };
```

Comment:
I'd drop this metric, we already report the same information as a counter.

Dito for `fabric.io.rate` below.

Reply:
Many plugins have a `StoreRates` option that will translate counters to a rate. The same can easily be added for the *write_log plugin*.

Reply:
> Many plugins have a `StoreRates` option that will translate counters to a rate.

So it seems:
```
$ git grep -l StoreRates src/write_*
src/write_graphite.c
src/write_http.c
src/write_influxdb_udp.c
src/write_kafka.c
src/write_mongodb.c
src/write_redis.c
src/write_riemann.c
src/write_sensu.c
src/write_syslog.c
src/write_tsdb.c
```

Unfortunately not `write_prometheus` which is really the only one I care about currently (in addition to Sysman plugin) does not support it.

> The same can easily be added for the _write_log plugin_.

`write_log` plugin is missing also other features making usable for local monitoring / debugging, discussed in #4177.